### PR TITLE
Support additional standard device and content tags

### PR DESCRIPTION
**Pull request by karpov-kir**
Related to: https://bitmovin.atlassian.net/browse/TA-2598.

- Add `additionalStandardTags` to `Metadata` of `updateContentMetadata`
- This property can be used to pass Conviva standard tags defined in the documentation https://pulse.conviva.com/learning-center/content/sensor_developer_center/sensor_integration/javascript/js_quick_integration.htm

The same is done in Android https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/108.

### CODE REVIEW

File: src/ts/ContentMetadataBuilder.ts [hunk 1]
```diff
@@ -129,15 +140,8 @@ export class ContentMetadataBuilder {
...
-  set custom(newValue: Metadata['custom']) {
-    this.metadata.custom = newValue;
-  }
-
-  get custom(): Metadata['custom'] {
-    return {
-      ...this.metadataOverrides.custom,
-      ...this.metadata.custom, // Keep our custom tags in case someone tries to override them
-    };
+  addToCustom(toAdd: Metadata['custom']) {
```
Comment (dweinber) [inline src/ts/ContentMetadataBuilder.ts:147]: Why this API change? 🤔 

Previously, the customer could do this easily themselves (with the help of the getter), but also decide to remove a key. I don't think this is still possible. But to be fair, I'm not sure if there's a use case for that or if Conviva likes that.

**Review by dweinber — COMMENTED** [review]
Looks good, but there's one question to clarify please. Also, please resolve the CHANGELOG conflict

File: src/ts/ContentMetadataBuilder.ts (hunk 1 above)
Comment (karpov-kir) [reply src/ts/ContentMetadataBuilder.ts:147]: It is not exposed to customers, this is our internal API only.

Comment (karpov-kir) [reply src/ts/ContentMetadataBuilder.ts:147]: This is what is exposed to customers https://github.com/bitmovin/bitmovin-player-web-samples/blob/main/react/bundled-player/package-based-with-conviva/src/App.tsx#L81 and this has not been changed, except for added `additionalStandardTags`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 - Send player info metadata (framework and framework version)
+- Add `additionalStandardTags` to `Metadata` of `updateContentMetadata` for [Conviva pre-defined video and content metadata](https://pulse.conviva.com/learning-center/content/sensor_developer_center/sensor_integration/javascript/js_quick_integration.htm)
```
Comment (dweinber) [inline CHANGELOG.md]: "Add" is already in the section name 🙂 

```suggestion
- `additionalStandardTags` to `Metadata` of `updateContentMetadata` for [Conviva pre-defined video and content metadata](https://pulse.conviva.com/learning-center/content/sensor_developer_center/sensor_integration/javascript/js_quick_integration.htm)
```

File: src/ts/ContentMetadataBuilder.ts (hunk 1 above)
Comment (dweinber) [reply src/ts/ContentMetadataBuilder.ts:147]: Yes, of course 🤦 I think my caffeine level is still too low this morning 😅

**Review by dweinber — APPROVED** [review]

**Pull request merged**